R/tt_toString.R

```diff
@@ -590,29 +590,11 @@ setMethod("get_formatted_cells", "ElementaryTable",
 #' @rdname gfc
 setMethod("get_formatted_cells", "TableRow",
           function(obj, shell = FALSE) {
-            default_format <- if (is.null(obj_format(obj))) "xx" else obj_format(obj)
-            format <- lapply(row_cells(obj), function(x) {
```

Comment:
Note the obj here is the _row_ object, NOT the cell object, important to preserve that behavior

Reply:
ahhhh, I see! I took a moment, but now I understand. Also `TableRow` could have a format definition... mmmh we need to talk about format hierarchies because it is sometimes hard to understand which format will be finally picked and from where